src/pineko/theory.py

```diff
@@ -11,6 +11,7 @@
 
 import eko
 import numpy as np
+import pineappl
```

Comment:
Here you're adding this `import`, but I don't see it used anywhere,

Am I missing something?

Reply:
I am using pineappl here 
https://github.com/N3PDF/pineko/blob/842a31793c9e5b71498dc6c748404f4abffee1e2/src/pineko/theory.py#L355

Reply:
Actually, I'm rather surprised Pylint didn't notify

Reply:
yes because this branch originates from sv_check where that change happened

Reply:
Can you (at least quickly) investigate why that branch passes all checks, even if we know it to be invalid?

Reply:
But is not invalid, why should it be?

Reply:
> But is not invalid, why should it be?

Either you need to import, or you don't need. However, @felixhekhorn explained the mystery: it was, it has simply been fixed in both branches afterwards (so the `import pineappl` appears in the "Files Changed" of this PR, and it is needed even if there is no new `pineappl.<something>` anywhere)